Userland/Libraries/LibWeb/CMakeLists.txt

```diff
@@ -390,7 +390,7 @@ add_custom_target(generate_PropertyID.h DEPENDS CSS/PropertyID.h)
 
 add_custom_command(
     OUTPUT CSS/PropertyID.cpp
-    COMMAND /bin/mkdir -p CSS
+    COMMAND /usr/bin/env -S mkdir -p CSS
```

Comment:
Can't we somehow use `cmake` to make directories instead of using `/usr/bin/env -S mkdir -p ...`?

Reply:
```cmake
add_custom_command(
    ...
    COMMAND ${CMAKE_COMMAND} -E make_directory CSS
    ...
)
```
https://stackoverflow.com/a/3702233

Reply:
Also, this didn't need `-S` since it is not a shebang.